src/rulesets/properties.ts

```diff
@@ -56,7 +56,7 @@ export const rules = {
   propertyKey: ({ bodyProperties }: SnykApiCheckDsl) => {
     bodyProperties.added.must("have snake case keys", ({ key }, context) => {
       // TODO: did not find a doc link for this
-      const snakeCase = /^[a-z]+(?:_[a-z]+)*$/g;
+      const snakeCase = /^[a-z]+(?:_[a-z\d]+)*$/g;
```

Comment:
small change

Reply:
Matchers within square brackets are called character classes. A character range is valid inside of a character class. So are the shorthands like `\d`. They're meant to be mixed. source https://www.regular-expressions.info/charclass.html